composer.json

```diff
@@ -1,15 +1,18 @@
 {
-    "name": "tilleuls/amazon-mws",
```

Comment:
😄

Reply:
sorry, the file composer.json is not part of the change, but I could not delete it from the pull.

I believed that the pull would only pick up changes until the pull creation.

I will reverse again.

Reply:
no problem :)